frontend/src/api/visitEndpoints.js

```diff
@@ -2,11 +2,14 @@
 
 export const getVisits = api => async () => await api.get("visits/")
 
-export const updateVisits = api => async (id, body) =>
+export const updateVisits = api => async (id, body) => {
```

Comment:
Was there a specific reason for this?

Reply:
So I can't remember why I made this change, and I think that is a good enough reason for undoing it.